tests/conftest.py

```diff
@@ -386,9 +409,12 @@ def create_lambda_function(
     memory_size,
     publish,
     region_name,
+    security_groups=None,
+    subnet_ids=None
```

Comment:
```suggestion
    subnets=None
```

What about renaming `subnet_ids` to `subnets` so it is consistent with `security_groups`?

Reply:
okey.